zhaquirks/philips/__init__.py

```diff
@@ -174,48 +197,133 @@ def handle_cluster_request(
         ] = None,
     ):
         """Handle the cluster command."""
-        _LOGGER.debug(
-            "PhilipsRemoteCluster - handle_cluster_request tsn: [%s] command id: %s - args: [%s]",
+        self.debug(
```

Comment:
Please also replace this with `_LOGGER.debug` for now.

Reply:
Sorry to be a pest about this, but I'd really like to see it go forward. @fholzer - did you see the request from @TheJulianJES to make the logger change?

Reply:
hey, yes saw it. Was waiting to see if more changes are needed. Was hoping to keep the number of commits for small changes like this low.

Reply:
I'll add this as a commit to your branch, so that part should be done.
```suggestion
        _LOGGER.debug(
```
(All commits will be squashed when merging the PR anyways)